Sources/PkgConfig/PkgConfig.swift

```diff
@@ -58,7 +64,7 @@ struct PkgConfig {
     }
     
     static func locatePCFile(name: String) throws -> String {
-        for path in (searchPaths + envSearchPaths) {
+        for path in (pkgConfigSearchPaths + searchPaths + envSearchPaths) {
```

Comment:
Yes, I guess it will be a super set. I think the same can be true for `envSearchPaths` right? 

Shall I remove duplicates from three of them?